torchrl/modules/planners/common.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import abc
+from typing import Optional
+
+import torch
+
+from torchrl.data.tensordict.tensordict import TensorDictBase
+from torchrl.envs import EnvBase
+from torchrl.modules import TensorDictModule
+
+__all__ = ["MPCPlannerBase"]
+
+
+class MPCPlannerBase(TensorDictModule, metaclass=abc.ABCMeta):
+    """
+    MPCPlannerBase Module. This is an abstract class and must be implemented by the user.
```

Comment:
you can just say it is an abstract class.
Also I would remove this from the headline. Or just say "MPCPlannerBase base module" which says it all

Reply:
Don't forget: no new line between `"""` and the next word (i'm in the process of correcting the whole lib for these so I'd appreciate if new PRs don't have that)